examples/question-answering/README.md

```diff
@@ -24,6 +24,11 @@ uses special features of those tokenizers. You can check if your favorite model
 of the script.
 
 The old version of this script can be found [here](https://github.com/huggingface/transformers/tree/master/examples/legacy/question-answering).
+
+`run_qa` allows you to fine-tune any model from our [hub](https://huggingface.co/models) on the SQUAD dataset or another question-answergin dataset of the Datasets library or your own csv/jsonlines files as long as they are structured the same way as SQUAD. You might need to tweak the data processing inside the script if your data is structured differently.
```

Comment:
```suggestion
`run_qa.py` allows you to fine-tune any model from our [hub](https://huggingface.co/models) on the SQUAD dataset or another question-answering dataset of the `datasets` library or your own csv/jsonlines files as long as they are structured the same way as SQUAD. You might need to tweak the data processing inside the script if your data is structured differently.
```

same as vvove - not on any model. perhaps "any model trained on squad"?

Reply:
It's pretty much any model here too, as long as they have a `ForQuestionAnswering` head.